src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/NotifyAttentionNeededLicencesService.kt

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.jobs
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.NotifyAttentionNeededLicence
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.repository.LicenceRepository
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.NotifyService
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonerSearchApiClient
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.writeCsv
+import java.time.LocalDate
+
+@Service
+class NotifyAttentionNeededLicencesService(
+  @Value("\${notify.attentionNeededLicences.emailAddress}") private val emailAddress: String,
+  private val licenceRepository: LicenceRepository,
+  private val prisonerSearchApiClient: PrisonerSearchApiClient,
+  private val notifyService: NotifyService,
+) {
+
+  @Transactional
+  fun notifyAttentionNeededLicences() {
+    val attentionNeededLicences = licenceRepository.getAttentionNeededLicences()
+    if (attentionNeededLicences.isEmpty()) {
+      return
```

Comment:
can we log a message saying there were no licences, and could we add a log message saying how many licences we emailed about below?